src/typeahead/test/typeahead.spec.js

```diff
@@ -895,6 +908,12 @@ describe('typeahead tests', function() {
       $timeout.flush();
       expect(dropdown.css('top')).toEqual('500px');
     });
+
+    it('should add a custom class to append-to-body popup', function() {
```

Comment:
I think this test begs the question, if the user uses `typeahead-append-to-body`, should the component remove this class from the body tag when the element is destroyed? I think it should here.